### DISCUSSION TITLE
feature request: edit main menu

### DISCUSSION THREAD

**Discussion by wilfz** [doublecmd/doublecmd#1141] [Suggestions]
Is it possible to edit the main menu (like total cimmanders .mnu file)?
Otherwise please include something like "Start" menu in totalcmd.
I know i can create button bars but I prefer textual menu items to more or less self-explaining buttons.

**Comment by Skif-off** [comment]
> I prefer textual menu items

DC supports menu and submenu(s), for example: ![toolbarinbar.png](http://doublecmd.github.io/doc/en/images/imgDC/toolbarinbar.png)

As I remember, DC has no limitation by submenus count.

**Comment by wilfz** [comment]
Not quite what I am looking for. I want to extend the main menu with items and submenu of my own, rather than to add a button with a submenu..

**Comment by majkinetor** [comment]
Start menu is definitely valuable option. I don't use button bar at all and hide it so the button solution is unusable for me.

**Comment by wilfz** [comment]
See similar request discussion under https://github.com/doublecmd/doublecmd/discussions/1320
